data/seeds/01-users.js

```diff
@@ -0,0 +1,9 @@
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('users')
+    .truncate()
+    .then(function() {
+      // Inserts seed entries
+      return knex('users')
```

Comment:
I assume there are/were plans to put in actual seed entries here (since the users table will be empty after the truncate)?